### PR TITLE
Add support for eth_getLogs

### PR DESCRIPTION
**Pull request by axic**
This is useful by testing frameworks such as Truffle.

cc @gnidan

### CODE REVIEW

**Comment by gnidan** [comment]
(fwd cc @cgewecke because I'm going to be on holiday next week)

Thanks for keeping us in the loop @axic! Glad to see this!

**Review by frozeman — APPROVED** [review]
I am surprised that we didn't expose this yet.

File: lib/web3/formatters.js [hunk 1]
```diff
@@ -112,6 +112,20 @@ var inputTransactionFormatter = function (options){
...
+ *
+ * @method inputGetLogsFormatter
+ * @param {Object} getLogs options
+ * @returns {Object}
+ */
+var inputGetLogsFormatter = function (options) {
+    if (options.fromBlock)
+        options.fromBlock = inputBlockNumberFormatter(options.fromBlock)
+    if (options.toBlock)
+        options.toBlock = inputBlockNumberFormatter(options.toBlock)
```
Comment (axic) [inline lib/web3/formatters.js:126]: I think these two were missing the `;` according to lint, sorry @frozeman for that.

**Comment by axic** [comment]
@gnidan @cgewecke this was merged, truffle could use it directly 🙌

**Pull request merged**